app/models/concerns/github_methods.rb

```diff
@@ -0,0 +1,25 @@
+module GithubMethods
+  extend ActiveSupport::Concern
+
+  class_methods do
+    def up_to_date?(params)
+      coerced_params = params.as_json
+      obj = find_by(
+        github_id: coerced_params[:github_id],
+        updated_on_github_at: coerced_params[:updated_on_github_at]
+      )
+      obj.present?
+    end
+
+    def update_or_create(params)
+      coerced_params = params.is_a?(Hash) ? params : params.as_json
```

Comment:
Просто
```suggestion
      coerced_params = params.to_hash
```

Відповідно вюди де я писав переіменувати на `to_params`, переіменовуєм на `to_hash`.